sys-testing/Cargo.toml

```diff
@@ -6,10 +6,13 @@ publish = false
 
 [features]
 default = ["aws-lc-sys"]
```

Comment:
Instead of setting a default feature here, could we just fallback to "aws-lc-sys" in the build script?

Reply:
As mentioned above, I don't think we can use that to pull in optional dependencies.